doc/contributing/docs/style.rst

```diff
@@ -73,22 +166,67 @@ aren't familiar with the
 or don't know the difference between them.
 For this reason, it's best to avoid using them.
 
-Specify link text
------------------
 
-When you provide a :doc:`link </contributing/docs/markup/links>`, clearly specify
-where it leads. In this way, you will not mislead the reader.
+    
+Spelling and punctuation
+------------------------
 
-Bad example:
+US vs British spelling
+~~~~~~~~~~~~~~~~~~~~~~
 
-    For more details, click :doc:`here </contributing/docs/markup/links>`.
+We use the US English spelling.
 
-    Use :doc:`this </contributing/docs/markup/links>`.
+Check your spelling and punctuation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Good example:
+Consider checking spelling, grammar, and punctuation with special tools.
 
-    For more details, refer to the documentation on
-    :doc:`making links </contributing/docs/markup/links>`.
+Ending punctuation in lists and tables
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    Use full :doc:`link names </contributing/docs/markup/links>`.
-    
\ No newline at end of file
+The following recommendations are *for the English language only.*
+You can find similar guidelines for the Russian language in the
+`external reference for Russian proofreaders <http://new.gramota.ru/spravka/letters/83-rubric-77>`__.
+
+Lists
+^^^^^
+
+In bulleted and numbered lists, end each list item with a period if:
+
+*   Any item forms a complete sentence when combined with the list introduction
+    that precedes the colon, *unless:*
+
+    -   All items have three or fewer words.
+
+    -   The items are UI labels, headings, subheadings, or strings.
+
+*   Any item by itself is a complete sentence.
+
+The above rules come from the
+`Microsoft style guide <https://docs.microsoft.com/en-us/style-guide/scannable-content/lists>`__.
+
+Please don't use commas or semicolons at the end of list items---it's
+either a period or nothing.
+Don't add redundant conjunctions like "and"/"or" before the last list item.
+General English punctuation rules still apply for text in lists.
+
+Tables
+^^^^^^
+
+For the text in cells, use periods or other end punctuation
+only if the cells contain complete sentences or a mixture of fragments and sentences.
+(This is also a
+`Microsoft guideline <https://docs.microsoft.com/en-us/style-guide/scannable-content/tables#punctuation>`__
+for the English language.)
+
+Besides, make sure that your table punctuation is consistent---either
+all similar list/table items end with a period or they all don't.
```

Comment:
'Your table punctuation' assumes a column here? Can we say that a punctuation is a column is the same as for a list?